main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+)
+
+func createPhoneBook(names []string, numbers []string) map[string]string {
+	m := make(map[string]string)
+	for i := range names {
+		m[names[i]] = numbers[i]
+	}
+	return m
+}
+
+func main() {
+	var count int
+	scanner := bufio.NewScanner(os.Stdin)
+	fmt.Println("How many contacts do you want to enter?")
+	scanner.Scan()
+	bcount := scanner.Text()
+	fmt.Sscan(bcount, &count)
```

Comment:
Here `fmt.Scan` would actually be more convenient, i.e.

```
fmt.Scan(&count)
```

Reply:
I use it to remove bug that occurs in Visual Studio terminal